docs/topics/db/examples/many_to_one.txt

```diff
@@ -77,7 +77,7 @@ Create an Article via the Reporter object::
 Create a new article, and add it to the article set::
 
     >>> new_article2 = Article(headline="Paul's story", pub_date=date(2006, 1, 17))
-    >>> r.article_set.add(new_article2)
+    >>> r.article_set.add(new_article2, bulk=False)
```

Comment:
I went with a slightly different solution in 9d7e2c7b447b2bbabe746770ebd26465cc564f05, otherwise we'd have to get into a discussion of `bulk=False` here.